amundsen_application/static/js/components/DashboardPage/ImagePreview/index.tsx

```diff
@@ -14,17 +15,43 @@ export interface ImagePreviewProps {
 interface ImagePreviewState {
   isLoading: boolean;
   hasError: boolean;
+  isModalVisible: boolean;
 }
 
-export class ImagePreview extends React.Component<ImagePreviewProps, ImagePreviewState> {
-  constructor(props) {
-    super(props);
+type PreviewModalProps = {
+  imageSrc: string,
+  onClose(): void,
+}
 
-    this.state = {
-      isLoading: true,
-      hasError: false,
-    }
-  }
+const PreviewModal = ({ imageSrc, onClose }: PreviewModalProps) => {
+  const [show, setShow] = React.useState(true);
+  const handleClose = () => {
+    setShow(false);
+    onClose();
+  };
+
+  return (
+    <Modal show={show} onHide={handleClose} scrollable="true" className="dashboard-preview-modal">
+      <Modal.Header closeButton={true}>
+        <Modal.Title className="text-center">Dashboard Preview</Modal.Title>
```

Comment:
nit: Can we put `Dashboard Preview` in the constants file? A while ago we started trying to make sure we pull out all hardcoded English text as we go along, for the purpose of being able to easily find all cases if and when we introduce support for translations. While you're in that file, mind removing `LOADING_TEXT` which is no longer used?

Reply:
Yeah! 
Totally agree, I will do the same on the next strings I found.